source/MainWindow.cpp

```diff
@@ -234,6 +237,24 @@ MainWindow::~MainWindow()
 	delete fUndoContext;
 }
 
+void
+MainWindow::AboutRequested()
+{
+	BAboutWindow* about = new BAboutWindow("ResourceEdit","x-vnd.Haiku-ResourceEdit");
```

Comment:
Needs to be "application/x-vnd.Haiku-ResourceEdit" or the about window won't find its icon.